ts/package.json

```diff
@@ -2,11 +2,12 @@
   "name": "grpc-web-client",
   "version": "0.6.3",
   "description": "gRPC-Web client for browsers (JS/TS)",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "dist/grpc-web-client.js",
+  "browser": "dist/grpc-web-client.umd.js",
+  "types": "dist/typings/index.d.ts",
   "scripts": {
     "clean": "rm -rf build && rm -rf dist",
```

Comment:
Nit: do we still need a `build/` folder? If not, let's remove it as part of this change.